src/Rand.cc

```diff
@@ -54,6 +54,9 @@ double Rand::DblUniform(double _min, double _max)
 //////////////////////////////////////////////////
 double Rand::DblNormal(double _mean, double _sigma)
 {
+  if (equal(_sigma, 0.0))
```

Comment:
The default epsilon for `equal` here is 1e-6, which means if the sigma is set to any smaller value, this branch would be taken. I suggest we either we do a direct comparison with `0.0` or set the epsilon value to [std::numeric_limits<T>::epsilon](https://en.cppreference.com/w/cpp/types/numeric_limits/epsilon)

Reply:
updated to use `std::numeric_limits::epsilon` . [33b900f](https://github.com/gazebosim/gz-math/pull/615/commits/33b900fa9c923760bd274b89b17ed0a165492707)